airtime_mvc/public/js/libs/angular.min.js

```diff
@@ -1,293 +1,5 @@
-/*
```

Comment:
What is this change?

Reply:
Removed //# sourceMappingURL=angular.min.js.map at the end of the file so that an error doesn't show up on some browsers.

Reply:
Ok. As this is an embedded upstream library, is that change specific to LT or has angular fixed it in a subsequent version?

Reply:
Just on LT.

Reply:
What is the error? Would a rebase on a newer version of Angular fix the problem?